### DISCUSSION TITLE
Could inline styles method take serialized css member?

### DISCUSSION THREAD

**Discussion by busynest** [lit/lit#2636] [Ideas]
`@state() member = [JSON CSS];`

`<button style=“this.member”>`

Could this example above work?
or maybe deconstruct to in-line styles? Just wondering.

**Comment by bathos** [comment]
You may be looking for [styleMap](https://lit.dev/docs/templates/directives/#stylemap) directive.

**Reply by busynest** [reply]
Cool, I’ll look into it. Thank you. I probably won’t use this in the end. Just thinking about things

**Reply by busynest** [reply]
So Sass is serialization meaning it evolved from (JSON CSS). Therefore, Lit's styleMap could replace Sass users.

**Comment by busynest** [comment]
`CSSResult` seems to be scoped properly. `CSSResultArray` does not scope, because it injects `<style>` tag into HTML `<head>` tag
